.ci/install_cc_env_ubuntu.sh

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+function check_err {
```

Comment:
I don't really understand - if you want to run this script separately you can but you are re-implementing a feature that is already in the shell. If you wanted to, you could add a command-line option/argument that conditionally called `set -e` I guess?